piknikki/rancid-tomatillos
# Refactor/css and html refactors

## What does this PR do?
 - I refactored to use a background image behind each profile so that I could use an overlay of a gradient. This enables the use of white text that stands out really well. I also added a footer and did a bunch of futzing around with spacing and style.
## How should it be tested?
 - Just check out the UI and make sure nothing is broken. Should be able to navigate with back/forward buttons on browser, as well as back button inside profile.
